tests/foreman/destructive/test_infoblox.py

```diff
@@ -16,21 +16,15 @@
 """
 import pytest
 
-from robottelo.hosts import get_sat_rhel_version
```

Comment:
Is get_sat_rhel_version removed?

Reply:
Also if we have different name for rhel 7 packages we should have it because you are cherry picking it for 6.11 ?

Reply:
No it's not, i think we shouldn't remove it, and for 6.11, I'm aware of the changes, I'll update that in cherrypick PR